homeworks/andrii.tkachenko_Hodr1337/6-async/async.js

```diff
@@ -0,0 +1,46 @@
+const tamagochi = {
+  health: 5,
+  enjoy: 5,
+  food: 5,
+};
+
+function healIt() {
```

Comment:
You need to do this like methods, not functions.

> Для зміни кожного показника має бути функція(метод), яка збільшує 1 показник на декілька поінтів, та зменшує інші показники на 1 (кожен).

The same for the following functions, which change the tamagotchi